docs/user-guide/suppressions.md

```diff
@@ -0,0 +1,80 @@
+# Bulk suppressions
+
+Enabling a rule as `"error"` can be difficult when an established code-base already contains many violations that cannot be auto-fixed.
+Stylelint lets you _suppress_ those legacy violations so the rule is enforced **only** for new code. You can then clear the backlog at your own pace.
+
+> [!IMPORTANT]
+> Only rules configured with severity `"error"` are suppressed.
+> Violations reported as `"warning"` are **not** suppressed.
+
+## Suppressions file
+
+The first time you run Stylelint with `--suppress-all` (or `--suppress-rule`), the tool creates a file called `stylelint-suppressions.json` in the current working directory.
+This JSON file records which rules are being ignored in which files, so that subsequent runs only report new problems.
+
+> [!TIP]
+> Commit the file. Keeping `stylelint-suppressions.json` in version control ensures every developer on the project shares the same baseline.
+
+## Creating the suppressions file
+
+After adding or promoting a rule to `"error"` in your configuration, capture the current violations and write a suppressions file with:
+
+```bash
+npx stylelint "**/*.css" --fix --suppress-all
+```
+
+This command:
+
+1. Creates or updates `stylelint-suppressions.json` (or the path given by `--suppressions-location`) at the project root.
+2. Removes auto-fixable problems (`--fix` is optional but recommended).
+3. Mutes every remaining violation so subsequent runs start clean.
+
+Commit the suppressions file so it’s shared with your team.
+
+## Targeting specific rules
+
+Want to mute only certain rules? Use `--suppress-rule` one or more times:
+
+```bash
+npx stylelint "**/*.css" --fix \
+  --suppress-rule block-no-empty \
+  --suppress-rule color-no-invalid-hex
+```
+
+Each flag adds or updates the counter for that rule in the suppressions file.
+
+### Changing the location
+
+You can keep the file elsewhere with `--suppressions-location`, but you **must supply the same flag every time you run Stylelint**, even during routine linting, so the tool can locate and maintain the file.
+
+```bash
+# Suppress existing errors and write the file into .github/
+npx stylelint "**/*.css" \
+  --suppress-all \
+  --suppressions-location .github/stylelint-suppressions.json
```

Comment:
minor

The folder used for the example should be generic/brand-less.
i.e. `./stylelint-suppressions.json` will do just fine

Also the example should show that the json file doesn't have to be named `stylelint-suppressions`.

Reply:
Good suggestions! I've fixed it.
[Adjust descriptions](https://github.com/stylelint/stylelint/pull/8564/commits/e4f09b35d868c9e2214ad6a0a0bd19561aa03e7a)